apps/calendar/elements/settings.html

```diff
@@ -14,11 +14,11 @@
         <div role="toolbar">
           <button class="settings toolbar-item">
             <span class="icon"
-                  data-l10n-id="advanced-settings-short">Settings</span>
+                  data-l10n-id="advanced-settings-short"></span>
```

Comment:
When updating strings we must change the key (rule imposed by l10n team) so for the two that you are changing lets have: advanced-settings-short-icon-button and drawer-sync-icon-button